core/templates/dev/head/pages/exploration_player/ConversationSkinDirective.js

```diff
@@ -482,11 +484,21 @@ oppia.directive('conversationSkin', [
           };
 
           $scope.submitAnswer = function(answer, interactionRulesService) {
-            // For some reason, answers are getting submitted twice when the
-            // submit button is clicked. This guards against that.
+            // Safety check to prevent double submissions from occurring.
+            // It's not clear if this is needed as the original
+            // issue may have possibly been resolved in PR#3474
+            // when redundant submitAnswer() calls were removed
+            // from TextInput and NumericInput
             if (_answerIsBeingProcessed ||
-                !$scope.isCurrentCardAtEndOfTranscript() ||
-                $scope.activeCard.destStateName) {
+              !$scope.isCurrentCardAtEndOfTranscript() ||
+              $scope.activeCard.destStateName) {
+              return;
+            }
+
+
+            if (!$scope.isInPreviewMode &&
+                FatigueDetectionService.addSubmission()) {
+              $scope.$broadcast('oppiaFeedbackAvailable');
```

Comment:
Without it, the submission text field and button don't ever come back, I think because it still thinks that it's waiting for a response.